app/src/main/res/layout/layout_main_title.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="30dp"
+    android:layout_marginEnd="30dp"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <TextView
+        android:textColor="@color/text_black"
+        android:id="@+id/subtitle_textview"
+        android:textSize="14sp"
+        android:fontFamily="@font/noto_sans_kr_regular"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:text="반갑습니다"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"/>
+
+    <TextView
+        android:textColor="@color/text_black"
+        android:text="사용할 기능을 선택하세요"
+        android:textSize="18sp"
+        android:fontFamily="@font/noto_sans_kr_bold"
+        android:id="@+id/title_textview"
+        app:layout_constraintTop_toBottomOf="@id/subtitle_textview"
+        app:layout_constraintStart_toStartOf="parent"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"/>
+
+    <ImageView
+        app:layout_constraintBottom_toBottomOf="@id/title_textview"
+        app:layout_constraintTop_toTopOf="@id/subtitle_textview"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:src="@drawable/splash_logo"
+        android:layout_width="45dp"
+        android:layout_height="45dp"/>
```

Comment:
`contentDescription` attribute is missing. Please consider adding it for better accessibility.